osm2gtfs/creators/routes_creator.py

```diff
@@ -13,4 +13,67 @@ def __repr__(self):
         return rep
 
     def add_routes_to_feed(self, feed, data):
-        raise NotImplementedError("Should have implemented this")
+        """
+        This function adds the routes from the data to the GTFS feed.
+        """
+        # Get route information
+        routes = data.get_routes()
```

Comment:
It is unclear if we are talking about OSM routes or GTFS routes. Shouldn't we call this var lines instead of routes ?

Reply:
Actually they contain both `Lines` (Route Masters) and `Itineraries` (OSM Routes), when they appear together in code they are just called `routes`.

Reply:
As I understand correctly, this variable only contains  `Lines` objects containing `Itineraries` objects. Wouldn't it be easier to understand by making this explicit with a `lines` variable ?